Add `Client/Interface/Chat console/Highlight` (#7403)
ppy/osu-wiki: wiki/Client/Interface/Chat_console/Highlight/en.md

---
tags:

>   - mention
>   - hilight

Reviewer: ```suggestion
  - mention
  - hilight
  - HL
  - ping
```

Author: huh, really

---

# Highlight

> **Highlight** (sometimes called a **mention**) is a way to draw someone's attention to the chat if they are online. By default, highlight is triggered when a player's username is written out in the chat. The list of additional highlight words may be customized in osu!'s [in-game chat options](/wiki/Client/Options#in-game-chat).

Reviewer: ```suggestion
A **highlight** (commonly abbreviated as **HL**, sometimes called a **mention** or **ping**) is a way to draw someone's attention to the chat if they are online. By default, a highlight is triggered when a player's username is written out in the chat. The list of what words get highlighted may be customised in osu!'s [in-game chat options](/wiki/Client/Options#in-game-chat).
```

Author: going to accept it for the sake of simplicity, but words aren't getting highlighted, the whole line does

It is possible to ignore highlights from specific users by adding their usernames to the [ignore list](/wiki/Client/Options/Ignore_list) with `@h` appended to them.

## Appearance

::: Infobox

> ![](img/hl-notification.png "An example of highlight notification")

Reviewer: ```suggestion
![](img/hl-notification.png "An example of a highlight notification")
```

:::

> When a chat message triggers highlight, the sender's username on that line becomes green, and the message is added to a `#highlight` chat tab. Additionally, if the osu!'s window is inactive, its icon will flash on the taskbar.

Reviewer: ```suggestion
When a chat message triggers highlight, the sender's username on that line becomes green, and the message is added to a `#highlight` chat tab. Additionally, if the osu! window is inactive, its icon will flash on the taskbar.
```
